source/contracts/reporting/DisputeBond.sol

```diff
@@ -51,7 +63,7 @@ contract DisputeBond is DelegationTarget, Extractable, ITyped, Initializable, Ow
         return true;
     }
 
-    function withdrawDisavowedTokens() public onlyOwner onlyInGoodTimes returns (bool) {
+    function withdrawDisavowedTokens() public onlyInGoodTimes returns (bool) {
```

Comment:
I took the owner requirement off as there is no harm I can perceive in someone else paying the gas to have the bond redeemed on behalf of whoever owns it when it is in disavowed state